python/tk_nuke_writenode/handler.py

```diff
@@ -35,7 +35,7 @@ class TankWriteNodeHandler(object):
     """
 
     SG_WRITE_NODE_CLASS = "WriteTank"
-    SG_WRITE_DEFAULT_NAME = "ShotgunWrite"
+    SG_WRITE_DEFAULT_NAME = "ShotGridWrite"
```

Comment:
Same reasoning here. Could this break compatibility with older scenes?

Reply:
This should be safe. It is only the default name we give to new writenodes